samples/drivers/led_is31fl3216a/README.rst

```diff
@@ -0,0 +1,36 @@
+.. _is31fl3216a:
+
+is31fl3216a: 16 channels PWD LEDs controller
+############################################
+
+Overview
+********
+
+This sample controls up to 16 LEDs connected to a is31fl3216a driver.
+
+Each LED is gradually pulsed until it reach 100% of luminosity and gradually
+turned off again.
+
+Test pattern
+============
+
+For each LED:
+- Increase the luminosity until 100% is reached
+- Decrease the luminosity until completely turned off
+
+Building and Running
+********************
+
+This sample can be built and executed on boards with an I2C is31fl3216a LED
+driver connected. A node matching the "issi,is31fl3216a" binding should be
+defined in the board DTS files.
```

Comment:
Replace with:

```
This sample can be built and executed when the devicetree has an I2C device node
with compatible :dtcompatible:`issi,is31fl3216a` enabled, along with the relevant
bus controller node also being enabled.
```

The reason why is that, as the following paragraph shows, the definition of the led device node does not have to be in a board file. It is also better to link to the compatible in the bindings index with `:dtcompatible:` so users know what they can do with these nodes.